DailyPlayground/src/components/Tray/Tray.tsx

```diff
@@ -48,6 +48,19 @@ export default function Tray({ disabled, onClickLeaveCall }: Props) {
     callObject?.setLocalAudio(isMicMuted);
   }, [callObject, isMicMuted]);
 
+  const startPlayback = useCallback(async () => {
+    const res = await callObject?.startRemoteMediaPlayer({
+      url: 'an mp4 url',
+      settings: {
+        state: 'play',
+      },
+    });
+    if (res?.session_id) {
+      callObject?.updateParticipant(res?.session_id, {
+        setSubscribedTracks: {rmpAudio: true, rmpVideo: true}
+      });
+    }
```

Comment:
Issues happen with and without this, not sure if we need to do this, playback works without